src/js/containers/agencyV2/awardSpending/SubAgencySummaryContainer.jsx

```diff
@@ -4,10 +4,10 @@ import { isCancel } from 'axios';
 import PropTypes from 'prop-types';
 
 import { InformationBoxes } from "data-transparency-ui";
-import { awardTypeGroupLabels, awardTypeGroups } from 'dataMapping/search/awardType';
-import { fetchSubagencyCount, fetchSubagencyNewAwardsCount, fetchSubagencySummary } from 'apis/agencyV2';
+import { awardTypeGroups } from 'dataMapping/search/awardType';
+import { fetchSubagencyNewAwardsCount, fetchSubagencySummary } from 'apis/agencyV2';
 import BaseAgencySubagencyCount from 'models/v2/agency/BaseAgencySubagencyCount';
-import { setSubagencyCount, resetSubagencyCount } from 'redux/actions/agencyV2/agencyV2Actions';
+import { resetSubagencyCount } from 'redux/actions/agencyV2/agencyV2Actions';
```

Comment:
Why do we need to reset a value that isn't being set?

Reply:
Missed a step in removing this request, thanks for catching! Updated now